spec/nine_one_one_spec.rb

```diff
@@ -171,33 +98,109 @@
         end.to raise_error(NineOneOne::ConfigurationError)
       end
     end
+
+    context 'multi config' do
```

Comment:
just fyi: http://www.betterspecs.org/#contexts i.e. I'd reanme it to 'when multi config'